src/WpfMath.Tests/OverUnderBoxTests.fs

```diff
@@ -32,3 +28,7 @@ type OverUnderBoxTests() =
     member _.``OverUnderBox rendering calls to RenderTransformed for delimiter``() =
         overUnderBox.RenderTo(mockedRenderer, x, y)
         Mock.Verify(<@ mockedRenderer.RenderTransformed(delimiterBox, any(), any(), any()) @>, once)
+
+    [<Fact>]
+    member _.``OverUnderBox should be successfully constructed a box for over == false and scriptBox == null``() =
```

Comment:
```suggestion
    member _.``OverUnderBox should be successfully constructed for over == false and scriptBox == null``() =
```

Reply:
Also, I'll replace `==` with `=`, since this is F#.